autoscheduler/frontend/src/components/SchedulingPage/SchedulePreview/ScheduleDetails/ScheduleDetails.css

```diff
@@ -0,0 +1,56 @@
+.previous-button {
+    position: absolute;
+    left: 0;
+    top: 50%;
+    transform: translate(-120%, -50%);
+}
+
+.next-button {
+    position: absolute;
+    right: 0;
+    top: 50%;
+    transform: translate(120%, -50%);
+}
+
+.icon-container {
+    margin: 5px 8px 0px 0px;
+}
+
+.schedule-info-container {
+    display: grid;
```

Comment:
I believe you could accomplish the same thing with a table, but I prefer grid since it only takes a single line to define the layouts and is generally more flexible